.github/workflows/release.yml

```diff
@@ -8,29 +8,20 @@ on:
 permissions:
   contents: write
   packages: write
+  id-token: write
```

Comment:
Just curious why this is needed.

Reply:
The [Sign](https://github.com/microsoft/digitalworkplace-workflows/blob/v3.1/.github/workflows/sign.yml#L46-L51) and [Publish](https://github.com/microsoft/digitalworkplace-workflows/blob/v3.1/.github/workflows/publish.yml#L48-L53) workflows login to Azure for access to a key vault and storage account. [Previously](https://github.com/microsoft/digitalworkplace-workflows/commit/09bdb9203f30ff4185185436e710f0a2593a7a4f#diff-02134701e1a1341310f8f0a1cfd47db2dadbe2e6413dfc475623c7c8a67ed6a5) we passed the `AZURE_CREDENTIALS`/`az_creds` secret for login; it must be updated when the client secret expires.

We're now using [OpenID Connect](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows#use-the-azure-login-action-with-openid-connect) federated credentials that eliminate the need for a client secret. GitHub's [documentation](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-azure#adding-permissions-settings) says that the `id-token: write` permission is required.